tools/appsre-ansible/rpmbuild.yml

```diff
@@ -29,6 +29,9 @@
 
   # RHEL mock templates don't work on RHUI, they use the CDN repos
   - name: Subscribe
+    register: result
+    retries: 5
+    until: result is success
```

Comment:
I think that line is not needed but would probably work

Reply:
…and then also `register: …` is not needed - `retries: 5` should actually be sufficient if we don't need to have a special condition other than "not failing"

Reply:
I think you need the register / until for the retries to work? Not sure.

Reply:
Seems to be "new" in v2.16
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met 
> If until is not specified, the task will retry until the task succeeds but at most `retries` times (New in version 2.16).